Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -162,20 +162,20 @@
                 "editorExtension": "ms.vss-services-azure.azurerg-parameters-grid"
             }
         },
-        {
-            "name": "deploymentMode",
-            "type": "pickList",
-            "label": "Deployment mode",
-            "defaultValue": "Incremental",
-            "required": true,
-            "groupName": "Template",
-            "helpMarkDown": "Refer to [this](https://docs.microsoft.com/en-us/azure/azure-resource-manager/deployment-modes) for more details. \n\n Incremental mode handles deployments as incremental updates to the resource group . It leaves unchanged resources that exist in the resource group but are not specified in the template. \n\n Complete mode deletes resources that are not in your template. \n [Warning] This action will delete all the existing resources in the resource group that are not specified in the template. \n\n Validate mode enables you to find problems with the template before creating actual resources. \n\n By default, Incremental mode is used.",
-            "options": {
-                "Incremental": "Incremental",
-                "Complete": "Complete",
-                "Validation": "Validation only"
-            }
-        },
+      {
+        "name": "deploymentMode",
+        "type": "pickList",
+        "label": "Deployment mode",
+        "defaultValue": "Incremental",
+        "required": true,
+        "groupName": "Template",
+        "helpMarkDown": "Refer to [this](https://docs.microsoft.com/en-us/azure/azure-resource-manager/deployment-modes) for more details. \n\n Incremental mode handles deployments as incremental updates to the resource group. It leaves unchanged resources that exist in the resource group but are not specified in the template. \n\n Complete mode deletes resources that are not in your template. Complete mode takes relatively more time than incremental mode. If the task times out, consider increasing the timeout, or changing the mode to 'Incremental'. \n [Warning] This action will delete all the existing resources in the resource group that are not specified in the template. \n\n Validate mode enables you to find problems with the template before creating actual resources. \n\n By default, Incremental mode is used.",
```

Comment:
Can you also run "node make.js build --task AzureResourceGroupDeploymentV2" This will auto-generate/update a resource file - which should also be checked in.